postqa/cli.py

```diff
@@ -87,10 +88,25 @@ def parse_args():
         default=False,
         action='store_true',
         help='Print the shimmed JSON rather than uploading it')
+
+    feature_parser = parser.add_mutually_exclusive_group(required=False)
+    feature_parser.add_argument(
+        '--probe-git',
+        dest='probe_git',
+        action='store_true')
+    feature_parser.add_argument(
+        '--no-probe-git',
+        dest='probe_git',
+        action='store_false')
```

Comment:
Do you need two switches.  Can you just make leaving `--probe-git` mean `--no-probe-git`?

Reply:
It isn't a requirement but I like being able to set an explicit flag regardless of what the default is.  This is a fairly common idiom.

Reply:
OK. Fine with me.